pkgs/love2d/example/assets/pocket_bomber/src/explosion.lua

```
@@ -0,0 +1,138 @@
+-- Explosion with cross propagation and damage timing
+
+local G = require("src.globals")
+local Colors = require("src.colors")
+local Grid = require("src.grid")
+local Utils = require("src.utils")
```

Luacheck (1.2.0)

[warning] unused variable 'Utils'

(W211)